extensions/wikia/StoriesLinkTag/StoriesLinkTagHelper.class.php

```diff
@@ -121,12 +121,21 @@ protected function getArticleDescription( $data ) {
 	 * @return string
 	 */
 	protected function getArticleImageUrl( $data ) {
-		if ( !empty( $data['_embedded']['wp:featuredmedia'][0]['media_details']['sizes']['medium_large']['source_url'] ) ) {
-			return $data['_embedded']['wp:featuredmedia'][0]['media_details']['sizes']['medium_large']['source_url'];
+		if ( empty( $data['_embedded']['wp:featuredmedia'][0] ) ) {
+			return '';
 		}
 
-		if ( !empty( $data['_embedded']['wp:featuredmedia'][0]['media_details']['sizes']['full']['source_url'] ) ) {
-			return $data['_embedded']['wp:featuredmedia'][0]['media_details']['sizes']['full']['source_url'];
+		$media = $data['_embedded']['wp:featuredmedia'][0];
+		if ( !empty( $media['media_details']['sizes']['medium_large']['source_url'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "media_details" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "source_url" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sizes" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)